platform/linux/linux.cmake

```diff
@@ -137,7 +137,9 @@ target_link_libraries(
 add_subdirectory(${PROJECT_SOURCE_DIR}/bin)
 add_subdirectory(${PROJECT_SOURCE_DIR}/expression-test)
 add_subdirectory(${PROJECT_SOURCE_DIR}/platform/glfw)
```

Comment:
Why do we include also `glfw` by default? I think this should also be excluded.

Reply:
Let's do that in a separate PR?

Reply:
OK 😊